### ISSUE TITLE
Difficult to move Element Desktop window on macOS

### ISSUE THREAD

**Issue by otri** [element-hq/element-web#32018]
### Steps to reproduce

1. Logged in to Element
2. Attempt to click-drag window to new position

#### What did you expect?
Should be able to grab the top 10pt top bar area, or anywhere there's empty space in the top window navigation bar areas.  For example the empty space between the "SPD" room name to the camera icon, should be grabbable area to move the window.

#### What happened instead?
Only a thin 2pt grab area is available to relocate the window.  Above that is a 5pt tall top resize bar, below does nothing.

<img width="1086" height="902" alt="Image" src="https://github.com/user-attachments/assets/1cd3119c-95fe-4bd8-a29c-96bed3abe970" />

### Operating system

macOS 15.6.1 (24G90)

### Application version

Element Version 1.12.0 (1.12.0)

### Homeserver

N/A. custom home server, running Synapse

### Will you send logs?

No

**Comment by t3chguy** [comment]
> For example the empty space between the "SPD" room name to the camera icon, should be grabbable area to move the window.

That whole area is a button, and in Electron something cannot be both clickable and a drag handle for the window

**Comment by florianduros** [comment]
> Only a thin 2pt grab area is available to relocate the window. Above that is a 5pt tall top resize bar, below does nothing.

The area to drag the window has the height of 13px. You can see the space above the selection:

<img width="1392" height="78" alt="Image" src="https://github.com/user-attachments/assets/d2efa9f4-6b6a-455f-a19b-cb3cddb0c02c" />

<img width="1392" height="78" alt="Image" src="https://github.com/user-attachments/assets/a36bde53-293f-445a-99a0-f739f598d996" />

**Comment by t3chguy** [comment]
Which is still tiny...

Drag area on Chrome is massive

<img width="1722" height="81" alt="Image" src="https://github.com/user-attachments/assets/bc3a9887-c391-4713-9826-932e2eccc20d" />

Drag area in JB IDEs still large

<img width="1729" height="51" alt="Image" src="https://github.com/user-attachments/assets/a36170a6-4016-4a4c-bfdb-d7ad8f12221d" />

Drag area in Spotify is huge

<img width="1753" height="95" alt="Image" src="https://github.com/user-attachments/assets/6cda8070-355a-47cd-abc3-6f5470cc17e7" />

Drag area in Beeper is narrow but tall

<img width="384" height="72" alt="Image" src="https://github.com/user-attachments/assets/b9fb4752-098d-4142-9870-9e2d1d501735" />

Drag area in Notion is similar to JB

<img width="1752" height="56" alt="Image" src="https://github.com/user-attachments/assets/a0ab01a5-4bc2-4ae3-ac39-1f717d2b9b5b" />

Drag area in WhatsApp is comprehensive

<img width="1046" height="64" alt="Image" src="https://github.com/user-attachments/assets/56403e72-4015-48ef-bc47-398b3129647b" />

**Comment by otri** [comment]
> > Only a thin 2pt grab area is available to relocate the window. Above that is a 5pt tall top resize bar, below does nothing.
> 
> The area to drag the window has the height of 13px. You can see the space above the selection:
> 
> <img alt="Image" width="1392" height="78" src="https://private-user-images.githubusercontent.com/2621378/493246375-d2efa9f4-6b6a-455f-a19b-cb3cddb0c02c.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTg3MjIwODcsIm5iZiI6MTc1ODcyMTc4NywicGF0aCI6Ii8yNjIxMzc4LzQ5MzI0NjM3NS1kMmVmYTlmNC02YjZhLTQ1NWYtYTE5Yi1jYjNjZGRiMGMwMmMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDkyNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTA5MjRUMTM0OTQ3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZmFjNTE5MTVmODgyYTA4MDMxNjIwNjgyMzZjMTAwOGM4ZWRjZjA0Njk5ZTI3ZmRlOWUzNWYwNjM5ZDU1NDJiZiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.fQa70fPQLlSNqeTFHW0MUVYZdeo-tqNPXlQ6GybLJuE"> <img alt="Image" width="1392" height="78" src="https://private-user-images.githubusercontent.com/2621378/493246559-a36bde53-293f-445a-99a0-f739f598d996.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTg3MjIwODcsIm5iZiI6MTc1ODcyMTc4NywicGF0aCI6Ii8yNjIxMzc4LzQ5MzI0NjU1OS1hMzZiZGU1My0yOTNmLTQ0NWEtOTlhMC1mNzM5ZjU5OGQ5OTYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDkyNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTA5MjRUMTM0OTQ3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MGFkZjdhOGE0NDU2OGQ0MTUzNGQ5YzQ3ZTM2M2U1OTJiMjYxYTU0Njk2NzhkNzI0OGEwMDI1ZDE0YzZjNDIxNCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.zl9o0SO-37Ec2PNXnfVJiWMtX0x5f4sj_9jB9Om4CLE">

Okay, so 13px is very tall on non-retina display. This top zone is effectively 13px / 2pts per pixel => 6.5 pts on a retina screen, and if you look at the actual drag-able areas, it's closer to 2pts, because of the approx ~5pt resize area above it.

\--
|  <-- 5pt resize window slice
\--
|  <-- 2pt drag window slice
\--
|
|  <-- Search or Room Name Zone
|
|
|
|
|
\--

The way Room name button toggle works has been a bit thorny since it was introduced ages ago.  I'm sure some people are used to the new functionality and prefer it, but I really liked the prior model of seeing a line of detail below the Room name.  I used this to link to a scratchpad document on whatever service I chose.  It was so handy to keep persistent things there, and use Matrix chat for ephemeral stuff.  This toggling button behaviour to reveal the sidebar is redundant to the (i) button on the far right.
